components/slices/MultipleUse.vue

```diff
@@ -133,6 +134,15 @@ export default {
       }
       return rows
     },
+    replaceBlockClass() {
+      const blockClass = document.getElementsByClassName('block')[0]
+      blockClass.className += ` ${this.slice.primary.block_style}`
```

Comment:
```suggestion
      blockClass.className += ` ${this.slice.primary.block_style}`
```

**Question :** Ou plutôt `blockClass.className += ' ' + this.slice.primary.block_style` pour plus de lisibilité quant à l'utilisation de l'espace dans l'ajout de classe ?